test/integration/admin/api/registry/policies_controller_test.rb

```diff
@@ -7,6 +7,8 @@ def setup
     @provider = FactoryBot.create(:provider_account)
     host! @provider.admin_domain
     @access_token = FactoryBot.create(:access_token, owner: @provider.admin_users.first!, scopes: %w[policy_registry], permission: 'rw').value
+    ::Logic::RollingUpdates.stubs(enabled?: true)
+    ::Account.any_instance.stubs(provider_can_use?: true)
```

Comment:
Probably better stubbing only for `@provider`.

Reply:
I think that it won't work because the controller "finds" the account from the access token, so it's another instance. But you can try 👍

Reply:
Right. Integration.